programs/bpf/c/sdk/scripts/install.sh

```diff
@@ -4,20 +4,51 @@ cd "$(dirname "$0")"/..
 
 # Install Criterion for all supported platforms
 version=v2.3.2
-(
-  [[ ! -d criterion-$version ]] || exit 0
-  set -ex
-  wget https://github.com/Snaipe/Criterion/releases/download/$version/criterion-$version-osx-x86_64.tar.bz2
-  wget https://github.com/Snaipe/Criterion/releases/download/$version/criterion-$version-linux-x86_64.tar.bz2
-  tar jxf criterion-$version-osx-x86_64.tar.bz2
-  tar jxf criterion-$version-linux-x86_64.tar.bz2
-  rm -rf criterion-$version-osx-x86_64.tar.bz2 criterion-$version-linux-x86_64.tar.bz2
+if [[ ! -d criterion-$version ]]
```

Comment:
`if [[ ! -d criterion-$version ]]; then` please.
"then" gets super lonely when it's relegated to the next line :)

Reply:
Such compassion!